symphony/lib/toolkit/class.administrationpage.php

```diff
@@ -1261,7 +1262,9 @@ public function addTimestampError($errorMessage, $existingObject, $options = arr
         ));
         $errdiv->appendChild(new XMLElement('label',
             Widget::Input('action[ignore-timestamp]', null, 'checkbox')->generate() . ' ' .
-            __('Override? Please understand that data loss can occur.')
+            __("I don't care about %s's changes, pretend it never existed.", array(
```

Comment:
Shouldn't this be "pretend they…" or "pretend these…"?

Anyway it sounds complicated. What about "I don't care about %s's changes. Save my version."